src/geventhttpclient/client.py

```diff
@@ -46,16 +46,18 @@ def from_url(cls, url, **kw):
             kw.pop('ssl_options', None)
         return cls(url.host, port=url.port, ssl=enable_ssl, **kw)
 
-    def __init__(self, host, port=None, headers={},
-            block_size=BLOCK_SIZE,
-            connection_timeout=ConnectionPool.DEFAULT_CONNECTION_TIMEOUT,
-            network_timeout=ConnectionPool.DEFAULT_NETWORK_TIMEOUT,
-            disable_ipv6=False,
-            concurrency=1,
-            ssl=False, ssl_options=None, ssl_context_factory=None,
-            insecure=False,
-            proxy_host=None, proxy_port=None, version=HTTP_11,
-            headers_type=Headers):
+    def __init__(self, host, port=None, headers=None,
+                 block_size=BLOCK_SIZE,
+                 connection_timeout=ConnectionPool.DEFAULT_CONNECTION_TIMEOUT,
+                 network_timeout=ConnectionPool.DEFAULT_NETWORK_TIMEOUT,
+                 disable_ipv6=False,
+                 concurrency=1,
+                 ssl=False, ssl_options=None, ssl_context_factory=None,
+                 insecure=False,
+                 proxy_host=None, proxy_port=None, version=HTTP_11,
+                 headers_type=Headers):
+        if headers is None:
```

Comment:
I don't get this change, what's the point ? I'm out of the python world for some time now...

Reply:
It is about this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
It is about this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
It is about this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
It is about this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
It is about this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
It's about this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments for `headers`

Reply:
It's about this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments for `headers`